migration/build.gradle

```diff
@@ -0,0 +1,9 @@
+plugins {
```

Comment:
This will be the module where we will test whether the previous version and the current version are compatible